boards/arm/nrf9160_pca10090/Kconfig.board

```diff
@@ -8,10 +8,8 @@ if SOC_NRF9160_SICA
 
 config BOARD_NRF9160_PCA10090
 	bool "nRF9160 PCA10090"
-	depends on !TRUSTED_EXECUTION_NONSECURE
```

Comment:
Does this work? Will adding `CONFIG_TRUSTED_EXECUTION_NONSECURE=n` to `nrf9160_pca10090_defconfig` (and similarly for the `..._ns` version) keep the configs the same?

Reply:
Not sure what you mean, the resulting .config is the same before and after this PR AFAICT.

Reply:
The defconfigs are already enabling NONSECURE (they are not disabling it though AFAICT, because disabled is the default).